HPXMLtoOpenStudio/resources/hvac_sizing.rb

```diff
@@ -3166,17 +3172,17 @@ def self.get_ventilation_data(hpxml_bldg)
     vent_mech_bal_tot = vent_fans_mech.select { |vent_mech| vent_mech.fan_type == HPXML::MechVentTypeBalanced }
     vent_mech_erv_hrv_tot = vent_fans_mech.select { |vent_mech| [HPXML::MechVentTypeERV, HPXML::MechVentTypeHRV].include? vent_mech.fan_type }
 
-    # Average in-unit CFMs (include recirculation from in unit CFMs for shared systems)
-    q_sup_tot = vent_mech_sup_tot.map { |vent_mech| vent_mech.average_total_unit_flow_rate }.sum(0.0)
-    q_exh_tot = vent_mech_exh_tot.map { |vent_mech| vent_mech.average_total_unit_flow_rate }.sum(0.0)
-    q_bal_tot = vent_mech_bal_tot.map { |vent_mech| vent_mech.average_total_unit_flow_rate }.sum(0.0)
-    q_erv_hrv_tot = vent_mech_erv_hrv_tot.map { |vent_mech| vent_mech.average_total_unit_flow_rate }.sum(0.0)
-    q_cfis_tot = vent_mech_cfis_tot.map { |vent_mech| vent_mech.average_total_unit_flow_rate }.sum(0.0)
+    # In-unit CFMs (include recirculation from in unit CFMs for shared systems)
+    q_sup_tot = vent_mech_sup_tot.map { |vent_mech| vent_mech.total_unit_flow_rate }.sum(0.0)
+    q_exh_tot = vent_mech_exh_tot.map { |vent_mech| vent_mech.total_unit_flow_rate }.sum(0.0)
+    q_bal_tot = vent_mech_bal_tot.map { |vent_mech| vent_mech.total_unit_flow_rate }.sum(0.0)
+    q_erv_hrv_tot = vent_mech_erv_hrv_tot.map { |vent_mech| vent_mech.total_unit_flow_rate }.sum(0.0)
+    q_cfis_tot = vent_mech_cfis_tot.map { |vent_mech| vent_mech.total_unit_flow_rate }.sum(0.0)
```

Comment:
@yzhou601 I'm seeing some diffs that are caused by these changes. I don't really understand why you were proposing these changes. I'm going to revert the changes and we can discuss this when you get back.

Reply:
Oh, I think I see now. The Bob Ross test has 50 cfm of outdoor air supplied through the HVAC system. In the HPXML file, we defined a CFIS system with 50 cfm, but because the default is to run the CFIS system for 8 hrs/day, we weren't actually introducing 50 cfm of outdoor air each hour. So we should really use 150 cfm at 8 hrs/day. I will revert the code changes and update our test files, and then we should be good.